client.go

```diff
@@ -290,40 +291,79 @@ func (c *Client) Read(ctx context.Context, recordID string) (*Record, error) {
 	return record, nil
 }
 
-// NewRecord creates a new record of the given content type.
-func (c *Client) NewRecord(recordType string) *Record {
-	return &Record{
+// Write writes a new encrypted record to the database. Returns the new record (with
+// the original, unencrypted data)
+func (c *Client) Write(ctx context.Context, recordType string, data *map[string]string, plain *map[string]string) (*Record, error) {
```

Comment:
I don't think you need to pass pointers to maps here, since maps are already reference types:

`func (c *Client) Write(ctx context.Context, recordType string, data, plain map[string]string) (*Record, error)`

You can still pass nil as a `map[string]string`, like you would with an `error`.